app/controllers/admin/invoices_controller.rb

```diff
@@ -6,4 +6,20 @@ def index
   def show
     @invoice = Invoice.find(params[:id])
   end
+
+  def update
```

Comment:
is there a reason to have two update methods here?

Reply:
whoops, removed it